apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/eobByPatientIdPaged.json

```diff
@@ -3728,11 +3728,18 @@
             "code" : "MMM"
           } ]
         },
+        "modifier" : [ {
+          "coding" : [ {
+            "system" : "https://bluebutton.cms.gov/resources/variables/rev_cntr_ndc_qty_qlfr_cd",
+            "code" : "ML",
+            "display" : "Milliliter"
+          } ]
+        } ],
         "locationAddress" : {
           "state" : "FL"
         },
         "quantity" : {
-          "value" : 0
+          "value" : 234.567
```

Comment:
Assuming this was taken from some real data example? Good reminder that quantity can be decimal values, which I had previously not really thought about before this issue.

Reply:
Yea, pretty much throw any inference from some of these column names out of your common sense bucket....pretty sure there is even a __cnt_ column that can be a decimal...